### DISCUSSION TITLE
Dual toolhead and AMS

### DISCUSSION THREAD

**Discussion by cf-mp** [Donkie/Spoolman#479] [Ideas]
Thank you for the great software.  I have been using spoolman for a few months now and its been great to keep track of what filaments we have left and when I need to order more.  One of the printers I use spoolman on is a Rat Rig V-Core 4.  They are going to start shipping their IDEX add-on in a few weeks for the printer.  IDEX adds a 2nd toolhead to the printer so you can either use its to copy/mirror the print you are already printing for overall faster prints, to print a different color as part of the print or a completely different type of filament for support use.  On top of that later this year they are going to be launching what they call MMU, a 4 filament swapping system similar to Bambu's AMS system.   Have you thought about possibly adding support into Spoolman for these type systems?   Thanks!

**Comment by Donkie** [comment]
Spoolman technically supports any kind of dual toolhead etc system. It's up to whatever software you use that does the tracking (e.g. Moonraker or OctoPrint) to support it.
